purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1976,9 +1990,24 @@ class Purchases internal constructor(
         var logHandler: LogHandler
             @Synchronized get() = currentLogHandler
             @Synchronized set(value) {
+                verboseLogHandler = verboseLogHandlerToLogHandler(value)
                 currentLogHandler = value
             }
 
+        /**
+         * Set a custom log handler for redirecting logs to your own logging system.
+         * Defaults to [android.util.Log].
+         *
+         * By default, this sends info, warning, and error messages.
+         * If you wish to receive Debug level messages, see [debugLogsEnabled].
+         */
+        @JvmStatic
+        var verboseLogHandler: VerboseLogHandler
```

Comment:
Hmm it feels a confusing to have both this and `logHandler`. I think we should strive for a more common setup, which would be that "verbose" is just another log level that would print even more info than DEBUG. I think having 2 different log systems, and allowing devs to overwrite both is only going to be confusing. 

If we really wanted to have 2 different log systems, I would avoid using "verbose" and maybe rename to `extraLogHandler` or something, so it's not confused with the commonly used "verbose" log level.

Reply:
Yeah this is my fault. I added `verboseLogHandler` in iOS, which made sense at the time. Recently I added a new log level, but now both things are confusing together :/

Reply:
That's ok! I guess my question is, do we need `verboseLogHandler` or just having the new log level is enough? I would vote for having the log level and not the `verboseLogHandler` but lmk if I'm missing any use cases. I'm assuming removing `verboseLogHandler` would require a major in iOS right?

In any case, I think we should try to move towards an easier to understand logging system. I would say, we can have different systems in Android and iOS for now and once we have to do a new major in iOS try to change it there? As for hybrids, we should study what we can do... Ideally we would implement the system we want to use in the hybrids but support the current iOS systems.

Reply:
Yeah, I agree with you. This is one of the reasons why I opened the PR before adding tests, since I was getting the feeling that things might change in iOS and we might not want to have the exact same thing.

Do you think we are going to lean towards having just the LogLevel? If that's the case we can just do the LogLevel in Android for now

Reply:
I think we should think about the use cases for this first but using the log level seems like it should be enough IMO yeah.

Reply:
Definitely the `verbose` `LogLevel` is much more useful. We haven't used the `verboseLogHandler` (I had a use for it when I added it but I forget what).

Reply:
Ok, so I will look into that one to bring it to Android. Thanks everyone! I think I can close this PR for now then